modules/dnn/src/op_inf_engine.cpp

```diff
@@ -45,13 +45,21 @@ infEngineWrappers(const std::vector<Ptr<BackendWrapper> >& ptrs)
 InfEngineBackendNet::InfEngineBackendNet() : netBuilder("")
 {
     hasNetOwner = false;
+#if INF_ENGINE_VER_MAJOR_LE(INF_ENGINE_RELEASE_2019R1)
     targetDevice = InferenceEngine::TargetDevice::eCPU;
+#else
+    device_name = "CPU";
+#endif
 }
 
 InfEngineBackendNet::InfEngineBackendNet(InferenceEngine::CNNNetwork& net) : netBuilder(""), cnn(net)
 {
     hasNetOwner = true;
+#if INF_ENGINE_VER_MAJOR_LE(INF_ENGINE_RELEASE_2019R1)
     targetDevice = InferenceEngine::TargetDevice::eCPU;
+#else
+    device_name = "CPU";
+#endif
```

Comment:
Can we use just text `device_name` and load plugins for 2018R5 and 2019R1 by name? Using `getPluginByName`.